client/plugins/utils.js

```diff
@@ -144,4 +185,26 @@ export function supplant(str, subs) {
       return typeof r === 'string' || typeof r === 'number' ? r : a
     }
   )
-}
\ No newline at end of file
+}
+
+function getNextValue(cronField, currentValue, maxValue) {
+  if (cronField === '*') {
+    return currentValue + 1 <= maxValue ? currentValue + 1 : 0;
```

Comment:
There is an incorrect next date because of this increment.

![image](https://user-images.githubusercontent.com/67830747/221712972-6da0175b-19cb-40b3-9aea-64830a4b1f29.png)

The correct next date is March 6th:

![image](https://user-images.githubusercontent.com/67830747/221713069-a05a4d1b-bb6f-4b0a-b61e-e99d8489c3cd.png)

I'm not sure a quick fix because for hour we only want to increment if not incrementing would cause the date to be in the past.

Reply:
Thanks for your guidance. At first glance, I thought to install an external library like [cron-parser](https://github.com/harrisiirak/cron-parser), but I wanted to avoid adding new libraries. 
I will check the code and validate the results against more cron job expression validators. Thanks.